Add documentation for GHC-94458 and GHC-05661 (#422)
haskellfoundation/error-message-index: message-index/messages/GHC-05661/index.md

---

> title: Cannot use qualified in pre and postpositive position

Reviewer: I don't understand "postpositive" here - what about:
```suggestion
title: Cannot use qualified both before and after the module
```
